src/test/java/stepDefinitions/NewAccountPageStepDefinitions.java

```diff
@@ -20,46 +27,39 @@ public NewAccountPageStepDefinitions() {
     @Given("^I should see registration page heading (.*)$")
     public void iShouldSeeRegistrationPageHeading(String heading) throws Throwable {
         String actualHeading = heading;
+        Thread.sleep(2000); //added for stability of test
```

Comment:
For stability, we are using waitForElement methods. Please read about them, and remove Thread.sleep() method.